src/js/api/fetchExercises.js

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+
+export const fetchExercises = async options => {
```

Comment:
There is already fetch api

Reply:
I'll change it later, I just created it before for simple rendering cards